packages/dds/task-manager/src/taskManager.ts

```diff
@@ -53,14 +53,6 @@ const snapshotFileName = "header";
  * It is still experimental and under development.  Please do try it out, but expect breaking changes in the future.
  *
  * @remarks
- * ### Creation
```

Comment:
I don't think we should remove this info, it's required for anyone not using the consumption-centric approach.

Reply:
Possible compromise for the near term -- mark one path as "beta?" It's somewhat disingenuous because they're both at roughly the same 'readiness level.'

Reply:
This is a great comment and I'm not quite sure where I stand. On one point we are making statements that our public docs are specifically focused on 3P. On the other hand I agree that we can't simply ignore and steamroll 1P.

What I want to do is capture the nuanced scenarios that currently exist when it comes to creation. The hard part is that if our docs are 3P focused currently and we are sharing workflows that are not **yet** available we introduce confusion and complexity. I'm not sure how to balance these two.